oaknational/Oak-Web-Application
# Feat/puil 865/remove quiz back button ab

### Job Story

**When** I am taking a quiz
**I want to not see a back button until I have completed the quiz**
**So that I can avoid losing my progress**



## How to test

1. Go to https://eu.posthog.com/project/124/insights/LQo6vhll note the numbers in the detailed results

2. Go to https://deploy-preview-2634--oak-web-application.netlify.thenational.academy/pupils/programmes/geography-primary-year-3/units/land-use-how-diverse-are-local-and-uk-landscapes/lessons/uk-human-and-physical-features/starter-quiz

Complete the quiz

3. Refesh the posthog page. The number of sessions should have gone up

4. Launch the posthog toolbar from https://eu.posthog.com/project/124/toolbar
(it's https://deploy-preview-2634--oak-web-application.netlify.thenational.academy)

5. Refresh the tab with quiz in it. On the toolbar feature flags you can select each of the 3 variants of the quiz and try them. 

<img width="567" alt="image" src="https://github.com/user-attachments/assets/60f4be50-acfe-4a01-9b7d-b42606c9744b">


### Acceptance criteria

- [x]  both versions implemented
- [x]  conditional offer via posthog A/B
- [x]  outcomes tracked in posthog
